xtask/Cargo.toml

```diff
@@ -8,12 +8,11 @@ publish = false
 anyhow = "1"
 env_logger = { version = "0.10.0", default-features = false }
 glob = "0.3.1"
-hlsl-snapshots = { path = "./hlsl-snapshots"}
+hlsl-snapshots = { path = "../hlsl-snapshots"}
 log = "0.4.17"
 pico-args = "0.5.0"
 shell-words = "1.1.0"
 which = "4.4.0"
 
 [workspace]
-
-members = [".", "./hlsl-snapshots"]
+members = ["."]
```

Comment:
I think this can now be removed.

Reply:
Indeed - removed this in 6a31ddf26472ec7f689e95349b631c06a1156399 now, thanks!

Reply:
Thanks! `[workspace]` could go as well, as it's now empty.

Reply:
> Thanks! [workspace] could go as well, as it's now empty.

@teoxoy Without having `[workspace]` in `xtask/Cargo.toml` we get this error:

```
error: current package believes it's in a workspace when it's not:
current:   [..]/naga/xtask/Cargo.toml
workspace: [..]/naga/Cargo.toml

this may be fixable by adding `xtask` to the `workspace.members` array of the manifest located at: /Users/fornwall/src/naga/Cargo.toml
Alternatively, to keep it out of the workspace, add the package to the `workspace.exclude` array, or add an empty `[workspace]` table to the package's manifest.
```

So if we want to remove the workspace entry in `xtask` we need to add it to the root `Cargo.toml` workspace (or exclude it explicitly, with `workspace.exclude`). Let me know if you want that!

Reply:
Ah, right. It's ok as it is since we need a separate lock file for it.